validation/src/main/scala/hmda/validation/rules/lar/validity/V300.scala

```diff
@@ -0,0 +1,66 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.census.CBSATractLookup
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.PredicateRegEx._
+
+object V300 extends EditCheck[LoanApplicationRegister] {
+
+  val cbsaTracts = CBSATractLookup.values
+
+  val validCombination = cbsaTracts.map { cbsa =>
+    (cbsa.geoIdMsa, cbsa.state, cbsa.county, cbsa.tractDecimal)
```

Comment:
I wanted to check my understanding. It looks like the difference between `validCombination` and `validMdCombination` is that the former uses MSA and the latter uses MD. Is that right?

If so, I think `validMsaCombination` would be a clearer name.